smoke_tests.sh

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env bash
 set -ex
 
+hostname | grep 'fresh-follower-node' && {
+    echo "This is a follower node without an elasticsearch leader. Smoke tests cannot be run."
+    exit 0
+}
+
 attempts=3
 delay=10
 
-# temporary, to get 18.04 formula branch passing. remove once upgraded. this delay is incorporated there
-sleep 20 
-
 retry ./smoke_tests_app.sh $attempts $delay
```

Comment:
I think both of these would be failing as even the PHP side would try to connect to a non-existent ElasticSearch

Reply:
yes indeed, it never gets to the elasticsearch set of smoketests as the app attempts to connect to elasticsearch even for the `/ping` endpoint